migrations/educandu-2022-10-18-01-remove-sourceType-from-audio-plugin.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable camelcase, no-await-in-loop, no-console */
+export default class Educandu_2022_10_18_01_remove_sourceType_from_audio_plugin {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async collectionUp(collectionName) {
+    const result = await this.db.collection(collectionName).updateMany(
+      {},
+      {
+        $unset: {
+          'sections.$[sectionElement].content.sourceType': null
```

Comment:
I see. Here:
"In the former version, the "accessible version" is just a path and not a full URL"
you meant to say "In the former version, the "persistable version" is just a path and not a full URL" though, right?

So you would suggest that we persist the "portable version" from now on? Currently we persist the "persistable version" aka URL (if external) or CDN path.

Reply:
Yes, sorry, you're right, I meant the "persistable version" is just a path. And yes, I would suggest to also use the "portable version" for persistence from now on, sorry for the confusion. Watcha think?

Reply:
I think it will simplify things even more and we should do it! 😁 
Would it be fine for you to do it in a subsequent ticket, for all plugins (as it needs a thorough migration script), **after** we first migrate the plugins to use the new input and drop `sourceType`? I feel it can get convoluted to do both things at the same time. So keeping the way of persisting as it used to be until the first round of changes is applied.